doc/introduction.txt

```diff
@@ -211,18 +211,12 @@ Here is the state of that vision as of November 15th, 2017 (after Theano 1.0.0):
 Contact us
 ==========
 
-Discussion about Theano takes place in the theano-dev_ and
-theano-users_ mailing lists. People interested in development of
-Theano should check the former, while the latter is reserved for
-issues that concern the end users.
-
+Discussion about Theano 
 Questions, comments, praise, criticism as well as bug reports should
-be submitted to these mailing lists.
+be submitted in form of an issue ticket tracker located at the following link pymc-theano-dev_
```

Comment:
"...submitted to the project's issue tracker on GitHub: pymc-theano-dev_"

We should also limit the accepted correspondence to questions and bug reports.

Reply:
Updated